test/models/subnet_test.rb

```diff
@@ -102,15 +102,16 @@ class Subnet::Test < Subnet; end
 
   test "should strip whitespace before save" do
     s = subnets(:one)
-    s.network = " 10.0.0.22   "
+    s.network = " 10.0.22.0   "
```

Comment:
Right now we have no test to normalize the network. You can also keep the changes smaller and keep using `network = " 10.0.0.22  "` and assert that it uses `10.0.0.0` as the network later on.

Reply:
Right, let me create an explicit test case.